Woof/WoofTests/Helper/EncodableExtensionTests/Encodable+AsDictionaryTests.swift

```diff
@@ -0,0 +1,104 @@
+import XCTest
+
+final class EncodableAsDictionaryTests: XCTestCase {
+    func testAsDictionaryConvertsModelWithOnlyStoredPropertyToDictionary() {
+        // Given
+        let object = ModelWithOnlyStoredProperty(name: "John Doe", age: 30)
+
+        // When
+        let dictionary = object.asDictionary()
+
+        // Then
+        XCTAssertEqual(dictionary?["name"] as? String, "John Doe")
+        XCTAssertEqual(dictionary?["age"] as? Int, 30)
+        XCTAssertEqual(dictionary?.count, 2)
+    }
+
+    func testAsDictionaryConvertsObjectThatContainsOnlyMethodToDictionary() throws {
+        // Given
+        let object = ModelWithOnlyFunction()
+
+        // When
+        let dictionary = object.asDictionary()
+
+        // Then
+        XCTAssertEqual(dictionary?.count, 0)
+    }
+
+    func testAsDictionaryConvertsObjectThatContainsOnlyComputedPropertyToDictionary() throws {
+        // Given
+        let object = ModelWithOnlyComputedProperty()
+
+        // When
+        let dictionary = object.asDictionary()
+
+        // Then
+        XCTAssertEqual(dictionary?.count, 0)
+    }
+
+    func testAsDictionaryConvertsObjectThatContainsMixedPropertiesAndMethodToDictionary() throws {
+        // Given
+        let object = ModelWithMixedFields(name: "John")
+
+        // When
+        let dictionary = object.asDictionary()
+
+        // Then
+        XCTAssertEqual(dictionary?["name"] as? String, "John")
+        XCTAssertEqual(dictionary?.count, 1)
+    }
+
+    func testSitterModelSuccessfullyTransformedToDictionary() {
+        // Given
```

Comment:
This suggestion is related with the suggestion below.
```suggestion
    func testSitterModelSuccessfullyTransformedToDictionary() throws {
        // Given
```

Reply:
Fixed